src/main/scala/mesosphere/marathon/api/forwarder/JavaUrlConnectionRequestForwarder.scala

```diff
@@ -105,7 +105,12 @@ class JavaUrlConnectionRequestForwarder(
   }
 
   private def copyRequestToConnection(leaderConnection: HttpURLConnection, request: HttpServletRequest): Try[Done] = Try {
-    leaderConnection.setRequestMethod(request.getMethod)
+    if (request.getMethod == "PATCH") { //workaround to overcome java restrictions, see https://bugs.openjdk.java.net/browse/JDK-7016595
+      leaderConnection.setRequestProperty("X-HTTP-Method-Override", "PATCH")
```

Comment:
Does jetty automatically deal with `"X-HTTP-Method-Override", "PATCH"`? Did you try it locally?